qiskit/pulse/schedule.py

```diff
@@ -236,7 +241,7 @@ def _mutable_shift(self,
     # pylint: disable=arguments-differ
     def insert(self,
                start_time: int,
-               schedule: ScheduleComponent,
+               schedule: 'Schedule',
```

Comment:
This and anywhere a `ScheduleComponent` is an input type should be `Union['Schedule', Instruction]`.

Reply:
Sure thing